Userland/Libraries/LibVideo/PlaybackManager.cpp

```diff
@@ -732,4 +690,46 @@ class PlaybackManager::StoppedStateHandler : public PlaybackManager::PlaybackSta
     PlaybackState get_state() const override { return PlaybackState::Stopped; }
 };
 
+DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> PlaybackManager::create(NonnullOwnPtr<Demuxer> demuxer)
```

Comment:
wait. why did this have to move to the end of the file?

Reply:
Because `make` now won't work without `SeekingStateHandler` definition and the class was defined later in the file (that's what was causing compile errors).

Reply:
Updated commit description with this info